eo-maven-plugin/src/test/java/org/eolang/maven/AssembleMojoTest.java

```diff
@@ -261,6 +261,28 @@ void assemblesSuccessfullyInOfflineMode(final Logs out, @TempDir final Path temp
         );
     }
 
+    @Test
+    @Disabled
+    void configuresChildParameters(@TempDir final Path temp) throws IOException {
+        final Map<String, Path> res = new FakeMaven(temp)
+            .withHelloWorld()
+            .with("trackOptimizationSteps", true)
+            .execute(AssembleMojo.class)
+            .result();
+        MatcherAssert.assertThat(
+            res,
```

Comment:
@Yanich96 let's add assertion message here and below

Reply:
I have done